docs/services/navigation/_index.md

```diff
@@ -99,7 +101,7 @@ The following attributes are available for `Navigation` services:
 
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
-| `store` | obj | **Required** | The configuration to use for the Navigation data storage. There are two parameters: `type` (`"memory"` or `"mongodb"`) and `config`. <br> Examples: `"store": [{"type": "memory"}]`, `"store": [{ "type": "mongodb", "config": {"uri": "mongodb://127.0.0.1:27017"}]` |
+| `store` | obj | **Required** | The configuration to use for the Navigation data storage. There are two parameters: `type` (`"memory"` or `"mongodb"`) and `config`. <br> Examples: <br> `"store": [{"type": "memory"}]` <br> `"store": [{ "type": "mongodb", "config": {"uri": "mongodb://127.0.0.1:27017"}]` |
```

Comment:
```suggestion
| `store` | obj | **Required** | The configuration object to use for the Navigation data storage. Either `[{"type": "memory"}]` or `[{ "type": "mongodb", "config": {"uri": "mongodb://127.0.0.1:27017"} }]`. |
```

The reason I think that's better is because for memory you don't need the second parameter.